_rules/target-size-ua-control-vcup8d.md

```diff
@@ -0,0 +1,136 @@
+
+This rule applies to any [HTML element][namespaced element] which [can be targeted by a pointer event][].
+
+## Expectation
+
+Each test target is a [User Agent controlled component][].
+
+## Assumptions
+
+- This rule assumes that [focusable][] `widget` are effectively clickable. If a widget is [focusable][] without being clickable, it may fail this rule while [Success Criterion 2.5.5 Target Size (enhanced)][sc255] and [Success Criterion 2.5.8 Target Size (minimum)][sc258] are satisfied.
+
+## Accessibility Support
+
+Hit testing isn't properly defined, and this has been an [issue in the CSS specification](https://github.com/w3c/csswg-drafts/issues/2325) for years. Therefore, different User Agents may perform it differently, resulting in different [clickable areas][clickable area] for the same element. As of February 2024, the ACT rules Community Group is not aware of actual cases resulting in significantly different [clickable areas][clickable area].
+
+## Background
+
+### Bibliography
+
+- [Understanding Success Criterion 2.5.5: Target Size (enhanced)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html)
+- [Understanding Success Criterion 2.5.8: Target Size (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This checkbox is an [User Agent controlled component][].
+
+```html
+<p id="accept">
+	<input aria-labelledby="accept" type="checkbox" />
+	I agree with the terms and conditions.
+</p>
+```
+
+### Failed
+
+#### Failed Example 1
+
+These radio buttons have their size modified by the author and are therefore not [User Agent controlled components][user agent controlled component]. Their [clickable area][] is too small.
+
+```html
+<style>
+	input[type='radio'] {
+		width: 1em;
+		height: 1em;
+	}
+</style>
+<fieldset>
+	<legend>Pick a color (required)</legend>
+	<label><input type="radio" name="color" value="blue" />Blue</label>
+	<label><input type="radio" name="color" value="yellow" />Yellow</label>
+</fieldset>
+```
+
+#### Failed Example 2
+
+This `button` is not a [User Agent controlled component][].
+
+```html
+<style>
+	#target {
+		width: 35px;
+		height: 35px;
+		border-radius: 0;
+	}
+</style>
+<button id="target" onclick="alert('hello')">Hi</button>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+These `input` elements and `button` are `disabled` and therefore not [focusable][].
+
+```html
+<fieldset disabled>
+	<label>First name <input /></label><br />
+	<label>Last name <input /></label><br />
+	<button>submit</button>
+</fieldset>
+```
+
+#### Inapplicable Example 2
+
+This button cannot be [targeted by a pointer event][] because it is entirely covered by the `div` element with a dashed red border.
+
+```html
+<head>
+	<title>Inapplicable Example</title>
+	<link rel="stylesheet" href="/test-assets/target-size/shared-styles.css" />
+	<style>
+		.cover {
+			top: 0;
+			height: 50px;
+			width: 500px;
```

Comment:
```suggestion
			min-height: 50px;
			min-width: 500px;
```

together with the CSS edit below, it ensures that if users zoom text only, the button is still covered by the red bordered box